src/orca/services/sevenbridges/hook.py

```diff
@@ -0,0 +1,48 @@
+from functools import cached_property
+
+from airflow.hooks.base import BaseHook
+from sevenbridges import Api
+
+from orca.services.sevenbridges.client_factory import SevenBridgesClientFactory
+from orca.services.sevenbridges.tasks import SevenBridgesTasks
+
+
+class SevenBridgesHook(BaseHook):
+    """Wrapper around SevenBridges client and tasks classes.
+
+    This hook was inspired by the Asana Airflow provider package:
+    https://github.com/apache/airflow/blob/main/airflow/providers/asana/hooks/asana.py
+    """
+
+    conn_name_attr = "sbg_conn_id"
+    default_conn_name = "sbg_default"
+    conn_type = "sbg"
+    hook_name = "SevenBridges"
+
+    def __init__(self, conn_id: str = default_conn_name, *args, **kwargs):
+        """Construct SevenBridgesHook using an Airflow connection.
+
+        Args:
+            conn_id: An Airflow connection ID.
+                Defaults to ``default_conn_name``.
+        """
+        super().__init__(*args, **kwargs)
+        self.connection = self.get_connection(conn_id)
+        extras = self.connection.extra_dejson
+        self.project = extras.get("project")
```

Comment:
In this case, I'm using the `project` name since that's what is used by SevenBridges. We would use `workspace` for Tower. I added the following to the description of the `hook.py` submodule. Let me know if this answers your questions. 

> This hook also takes advantage of
> extra field associated with connection objects to configure the
> underlying client and/or tasks objects. Whether to include a value
> as the extra connection field probably depends on whether that
> value is useful to change the overall state of the underlying
> client and/or tasks objects (e.g., `project` for SevenBridgesTasks)
> and whether the value is invariable for a given DAG.

Reply:
I think this is good, thanks!